maro/rl/rollout/env_sampler.py

```diff
@@ -155,15 +155,15 @@ def agent_names(self) -> list:
     def num_agents(self) -> int:
         return len(self.agent_state_dict)
 
-    def split_contents(self, agent2trainer: Dict[Any, str]) -> Dict[str, ExpElement]:
-        """Split the ExpElement's contents by trainer.
+    def split_contents(self, agent_to_algo_inst: Dict[Any, str]) -> Dict[str, ExpElement]:
```

Comment:
Is there any other mappings with name not following `aaa2bbb` ?
If not, I think maybe it is better to align the naming pattern, e.g. `agent2algoInst`/`agent2algoTrainer` ?

Reply:
It is not recommended to use upper class letters in the Python variables. Just change it to `agent2algoinst` anyway.